forte/sparse_ci/sq_operator_string.h

```diff
@@ -63,6 +64,9 @@ namespace forte {
  */
 using op_tuple_t = std::vector<std::tuple<bool, bool, int>>;
 
+// A class enum to encode if two operators commute, anti-commute or do not commute
+enum class CommutatorType { Commute, AntiCommute, MayNotCommute };
```

Comment:
Does `MayNotCommute` mean there's no commutation structure, or we are non-commital on it?

Reply:
It just means that we are non-committal on it. I added some comments to the code.